### DISCUSSION TITLE
Orientation of attiny1616

### DISCUSSION THREAD

**Discussion by jrhbcn** [PricelessToolkit/MailBoxGuard#20] [Q&A] [answered]
Hi, 

Finally I have managed to get hold of all components and I am trying to hand-solder them using a DIY solder plate. I am unsure of the placement of the attiny1616 chip. 

Right now I have soldered it with the dot in the top-left corner:

![db68k10srcdjtc4ejhagug60pv](https://github.com/PricelessToolkit/MailBoxGuard/assets/22198270/190fc6c3-12cc-4ffe-b5d0-9c7647e86f05)

However, I am not sure this is correct as my "light" multimeter probes seem to not detect correct ground/vcc as the [datasheet](https://datasheet.lcsc.com/lcsc/2004232205_Microchip-Tech-ATTINY1616-MNR_C507118.pdf) suggests:

Can anyone confirm the correct placement? Any tips on how to solder this vqfn component? Pads are extremelly small and I am struggling to see if it is correctly soldered.

Regards.

**Comment by PricelessToolkit** [comment, marked as answer]
Hi, nice board! The first pin is the "top right" See "MailBox_Guard_HTML_Boom.html" and don't forget to check "Highlight first pin"

**Comment by jrhbcn** [comment]
Argg, many thanks! Did not know that option. Unfortunately it was indeed in a bad orientation and I destroyed the pads when de-soldering  :(  I will have to de-solder the rest and use a new PCB.

Thank you again for releasing the project, I will let you know when I manage to make it work :)
